.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,35 @@
+name: dsp-appsec-trivy
+on: [pull_request]
+
+jobs:
+  appsec-trivy:
+    # Parse Dockerfile and build, scan image if a "blessed" base image is not used
+    name: DSP AppSec Trivy check
+    runs-on: ubuntu-latest
+    env:
+      # only used to name the image for this scan; it is not pushed anywhere
+      image: databiosphere/terra-external-credentials-manager
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Gradle cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: v1-${{ runner.os }}-gradle-${{ github.ref }}-${{ github.sha }}
+          restore-keys: v1-${{ runner.os }}-gradle-${{ github.ref }}
+
+      - name: Build image locally with Jib
+        run: |
+          ./gradlew :service:jibDockerBuild --image="${image}" -Djib.console=plain
```

Comment:
I'd expect this to fail, because it doesn't have the base image to build from. See https://github.com/DataBiosphere/terra-external-credentials-manager/blob/cc4dcee38c83319c7d1817a4e0b6d8fe6305c6c4/.github/workflows/publish.yml#L51-L65

Reply:
Could you clarify? The builds appear to succeed..

Reply:
Ah, it must just be building from the default base image, so it might still complete, but it wouldn't be scanning the actual produced image we'd be using.

If just running before an image is pushed is good enough, though, then we're already doing it and you can close this PR if you want.

Reply:
OK cool, yes that was the intent of this PR as well (running before pushing is exactly what we want).